Makefile

```diff
@@ -26,6 +27,7 @@ test-integration: venv
 	CACHI2_GENERATE_TEST_DATA=$(GENERATE_TEST_DATA) \
 	CACHI2_TEST_LOCAL_PYPISERVER=$(TEST_LOCAL_PYPISERVER) \
 		venv/bin/tox -e integration -- $(TOX_ARGS)
+	CACHI2_TEST_LOCAL_DNF_SERVER=$(TEST_LOCAL_DNF_SERVER)
```

Comment:
You'll need to move this variable one line above, this won't work as expected otherwise ;)